zgw/objecten-api/src/test/kotlin/com/ritense/objectenapi/ObjectenApiPluginFactoryTest.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.objectenapi
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.node.ObjectNode
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import com.ritense.plugin.domain.PluginConfiguration
+import com.ritense.plugin.domain.PluginConfigurationId
+import com.ritense.plugin.domain.PluginDefinition
+import com.ritense.plugin.domain.PluginProperty
+import com.ritense.plugin.service.PluginService
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+internal class ObjectenApiPluginFactoryTest {
+
+    @Test
+    fun `should create ObjectenApiPlugin`() {
```

Comment:
Not sure how I feel about this. This actually tests the PluginFactory more than the ObjectenApiPluginFactory

Reply:
I don't want to say "this is how all the other zgw plugin api's test their factories" but that is the case